_posts/2024-02-01-testing-with-wildfly-glow.adoc

```diff
@@ -0,0 +1,180 @@
+
+An example of output:
+
+----
+...
+layers inclusion rules
+* ee-core-profile-server
+  - BASE_LAYER
+* ee-concurrency
+  - JAVA_TYPE: [jakarta.enterprise.concurrent.*]
+* undertow-https
+  - ADD_ON
+...
+----
+
+This output means:
+
+* `ee-core-profile-server` is a base layer (always included).
+* `ee-concurrency` layer is included because a Java class located in the java package `jakarta.enterprise.concurrent` has been found.
+* `undertow-https` is included because it is bound to an included add-on (in this case `ssl`).
+
+BTW: The link:https://github.com/wildfly/wildfly/tree/main/testsuite/integration[WildFly integration tests] contain 
+a lot of examples of WildFly Glow scanning executions that you could use as a starting-point.
+
+=== Simple pom.xml example
+
+In this example, both the `wildfly-glow-arquillian-plugin` and `wildfly-maven-plugin` are bound to the 
+`test-compile` phase (run after the test classes have been compiled).
+
+The `wildfly-glow-arquillian-plugin` execution produces the file `target/glow-scan/provisioning.xml` that is then consumed by 
+the `wildfly-maven-plugin` to provision the server (in the directory `target/server`).
+
+[source,xml,subs=attributes+]
+----
+...
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.wildfly.glow</groupId>
+            <artifactId>wildfly-glow-arquillian-plugin</artifactId>
+            <version>1.0.0.Beta5</version>
+            <executions>
+                <execution>
+                    <id>scan</id>
+                    <goals>
+                        <goal>scan</goal>
+                    </goals>
+                    <phase>test-compile</phase>
+                </execution>
+            </executions>
+        </plugin>
+        <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-maven-plugin</artifactId>
+            <version>5.0.0.Beta2</version>
+            <configuration>
+                <provisioning-file>${project.build.directory}/glow-scan/provisioning.xml</provisioning-file>
+            </configuration>
+            <executions>
+                <execution>
+                    <id>test-provisioning</id>
+                    <goals>
+                        <goal>provision</goal>
+                    </goals>
+                    <phase>test-compile</phase>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+</build>
+----
+
+The provisioned server is then started by Arquillian and tests are executed. As you can see the provisioning aspects are fully handled by 
+the tooling, allowing you to focus on the testing.
+
+For more advanced usage (e.g.: selecting a WildFly version, an execution profile, adding `add-ons`) you should check the 
+WildFly Glow https://docs.wildfly.org/wildfly-glow[documentation].
+
+
+We hope that you will see an interest in using WildFly Glow in your WildFly application testing. 
```

Comment:
Not sure if I got the intent totally right here, but I think the following sounds better:

We hope that you will see how useful WildFly Glow can be for your WildFly application testing.